packet_test.go

```diff
@@ -85,4 +85,32 @@ func TestPacket(t *testing.T) {
 	b = []byte{}
 	require.NoError(t, pkt6.FromData(b))
 	require.Equal(t, b, pkt6.Data())
+
+	t.Run("CopyProperties", func(t *testing.T) {
```

Comment:
Can you avoid using the `t.Run()` pattern and do this instead (make sure to have no new line inside each group of tests inside each `t.Run()`):
```go
pkt7 := AllocPacket()
require.NotNil(t, pkt7)
defer pkt7.Free()
...
```

Reply:
Fixed.